### PR TITLE
Added dkim size

### PR DESCRIPTION
**Pull request by Pavlico**

### CODE REVIEW

**Comment by Pavlico** [comment]
#813

**Comment by Pavlico** [comment]
@DavidGarciaCat can you look at it :)

File: src/Api/Domain.php [hunk 1]
```diff
@@ -84,12 +86,13 @@ public function show(string $domain)
...
      * @param string[] $ips                an array of ips to be assigned to the domain
-     * @param ?string  $pool_id            pool id to assign to the domain
+     * @param ?string  $poolId             pool id to assign to the domain
      * @param string   $webScheme          `http` or `https` - set your open, click and unsubscribe URLs to use http or https. The default is http
+     * @param string   $dkimKeySize        Set length of your domain’s generated DKIM key
      *
      * @return CreateResponse|array|ResponseInterface
      */
-    public function create(string $domain, string $smtpPass = null, string $spamAction = null, bool $wildcard = null, bool $forceDkimAuthority = null, ?array $ips = null, ?string $pool_id = null, string $webScheme = 'http')
+    public function create(string $domain, string $smtpPass = null, string $spamAction = null, bool $wildcard = null, bool $forceDkimAuthority = null, ?array $ips = null, ?string $poolId = null, string $webScheme = 'http', string $dkimKeySize = '1024')
```
Comment (faizanakram99) [inline src/Api/Domain.php:95]: This would be a breaking change for people using named parameters

Comment (Pavlico) [reply src/Api/Domain.php:95]: i see then i guess i will move back underscore

Comment (faizanakram99) [reply src/Api/Domain.php]: I am not sure if named parameters are part of BC promise, maintainers of this project will know. But I wouldn't make any code style changes in a feature PR

**Review by DavidGarciaCat — APPROVED** [review]
Thanks!

**Comment by faizanakram99** [comment]
@DavidGarciaCat 

Any release coming soon ?

**Pull request merged**